cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,39 @@
-/// <reference types='cypress' />
+import { faker } from '@faker-js/faker';
+import HomeAndCataloguePageObject
+  from '../support/pages/homeCatalogue.pageObject';
 
-describe('', () => {
-  before(() => {
+const homeCatalogue = new HomeAndCataloguePageObject();
+const name = faker.person.firstName();
+const country = faker.location.country();
+const city = faker.location.city();
+const card = faker.finance.creditCardNumber();
+const month = faker.date.month();
+const year = new Date().getFullYear();
 
+describe('Demoblaze', () => {
+  before(() => {
+    homeCatalogue.visit();
   });
 
-  it('', () => {
+  it('should add new product in cart', () => {
+    homeCatalogue.clickOnCategory('Laptops');
+    homeCatalogue.clickOnProduct('Sony vaio i7');
+    homeCatalogue.clickOnButton('Add to cart');
+
+    homeCatalogue.verifyAddingToCart('Product added');
+
+    homeCatalogue.clickOnLink('Cart');
+    homeCatalogue.verifyProductinCart('Sony vaio i7');
```

Comment:
The method `verifyProductinCart` is being called. Double-check the spelling of the method name and ensure it matches the actual method name in the `HomeAndCataloguePageObject` class. It should be `verifyProductInCart` if following camelCase conventions.